.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -239,14 +264,14 @@ jobs:
           kayobe seed container image build $args
         env:
           KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
-        if: inputs.seed
+        if: inputs.seed && matrix.distro.arch == 'amd64'
```

Comment:
I think I asked this before but forgot the answer, why don't we build bifrost for arm as well?